github/resource_github_membership_test.go

```diff
@@ -2,29 +2,52 @@ package github
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"testing"
+	"unicode"
 
 	"github.com/google/go-github/v25/github"
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/terraform"
 )
 
 func TestAccGithubMembership_basic(t *testing.T) {
+	if testCollaborator == "" {
+		t.Skip("Skipping because length of `GITHUB_TEST_COLLABORATOR` is not set")
+	}
+
 	var membership github.Membership
+	var otherMembership github.Membership
+
+	oc := []rune(testCollaborator)
+	if unicode.IsUpper(oc[0]) {
```

Comment:
not sure if its possible, but could `0` ever be a number or symbol in a username?